include/mbedtls/asn1write.h

```diff
@@ -260,7 +260,7 @@ int mbedtls_asn1_write_printable_string( unsigned char **p,
  * \return          The number of bytes written to \p p on success.
  * \return          A negative error code on failure.
  */
-int mbedtls_asn1_write_utf8_string( unsigned char **p, unsigned char *start,
+int mbedtls_asn1_write_utf8_string( unsigned char **p, const unsigned char *start,
                                     const char *text, size_t text_len );
```

Comment:
I don't consider it in scope for this PR but I noticed that `mbedtls_asn1_write_utf8_string` and `mbedtls_asn1_write_printable_string` are defined the other way around in order when compared to the declarations in the header.

This is a very minor point but it would be a good quick fix to make it easier to read and maintain the codebase.

Reply:
Note: I would suggest changing the definitions (.c file) to match the declarations (.h file) and **not** the other way around.